.gitignore

```diff
@@ -63,27 +63,29 @@ WordPress/Derived Sources/
 WordPress/Images.xcassets/AppIcon-Internal.appiconset
 
 # Bundler
-/vendor/bundle/
-.bundle/
+.bundle
 
 # Dependencies
-/vendor/
+vendor
 
 # Fastlane
-Scripts/fastlane/Preview.html
-Scripts/fastlane/report.xml
-Scripts/fastlane/README.md
-Scripts/Preview.html
-Scripts/fastlane/test_output/
-Scripts/fastlane/google_cloud_keys.json
-Scripts/fastlane/promo_screenshots
-/Scripts/fastlane/metadata/review_information
-/Scripts/default.profraw
+fastlane/Preview.html
+fastlane/report.xml
+fastlane/README.md
+Preview.html
+fastlane/test_output/
```

Comment:
The directory was commited to git before you fixed the `.gitignore` path, so this PR currently have the `test-output/report.{html,junit}` files

Reply:
Fixed in d9e55ae027c828c2be41dade8a11ced887546638, thanks!